app/src/main/java/com/twilio/video/app/participant/ParticipantViewState.kt

```diff
@@ -1,23 +1,42 @@
 package com.twilio.video.app.participant
 
-import com.twilio.video.RemoteParticipant
+import com.twilio.video.LocalVideoTrack
+import com.twilio.video.NetworkQualityLevel
+import com.twilio.video.NetworkQualityLevel.NETWORK_QUALITY_LEVEL_UNKNOWN
+import com.twilio.video.Participant
 import com.twilio.video.VideoTrack
 
 data class ParticipantViewState(
-    val sid: String? = null,
-    val identity: String? = null,
+    val sid: String,
+    val identity: String,
     val videoTrack: VideoTrack? = null,
```

Comment:
Yeah basically you would have two ParticipantViewState objects. @timrozum showed me how he has it setup on iOS and he adds another screenTrack property so that only one participant object is needed. I'll push up a fix for this.